rampwf/score_types/detection.py

```diff
@@ -652,6 +647,79 @@ def precision_recall_curve(y_true, y_pred, conf_thresholds, iou_threshold=0.5):
     return np.array(ps), np.array(rs)
 
 
+def _add_id(y):
+    """Helper function to flatten and add id column to list of lists"""
+    y_new = []
```

Comment:
Mention here that this method removes empty lists instead of indexing them.

Reply:
done